.github/workflows/main.yml

```diff
@@ -21,7 +21,7 @@ defaults:
     shell: bash
 
 env:
-  BITCOIN_VERSION: "23.0"
+  BITCOIN_VERSION: "26.0"
```

Comment:
The way we have [this job](https://github.com/lightninglabs/docker-bitcoin-core/blob/0132c0cf3974546a3f71d81c86ebe52bc5414abd/.github/workflows/build.yaml) setup, leads to images with tags like `bitcoin-core:26` and `bitcoin-core:26-alpine`. I think we may have to either drop the minor version here or re-tag and push images with a new tag that includes the minor version.

Reply:
Ah, good point. Updated.